### PR TITLE
Initial Commit for Security Section in Handbook

### PR DESCRIPTION
**Pull request by DSchalla**
#### Summary

This pull request adds a security operations section to the handbook and is still work in progress. It will contain a reference to existing policies and processes, and explain the role from the different teams.

@jasonblais FYI for visibility, I'll assign you and Justine for review once it's ready for review.

### CODE REVIEW

**Comment by DSchalla** [comment]
@jasonblais @justinegeffen The two privacy pages and plugin security page we will commit separately, I'd like to get the structure published that we have the policy + user guide reference available for staff already. Happy to comment out empty sections in the menu until they are ready, however you prefer.

File: operations/security/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Security
+
+## Where to find us
+
+Mattermost: [Security Team Channel](https://community-daily.mattermost.com/private-core/channels/security-team)
```
Comment (jasonblais) [inline operations/security/README.md]:
```suggestion
Mattermost: [Security Team Channel](https://community.mattermost.com/private-core/channels/security-team)
```

File: operations/security/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Email: 
+* security@mattermost.com (General Inquiry)
+* security-alerting@mattermost.com (Incident Reporting / Alerting)
+* privacy@mattermost.com (Privacy related requests (e.g. GDPR, CCPA))
+* responsibledisclosure@mattermost.com (Responsible Disclosure)
```
Comment (jasonblais) [inline operations/security/README.md]:
```suggestion
* responsibledisclosure@mattermost.com ([Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/))
```

**Review by jasonblais — DISMISSED** [review]
Thanks! This seems like a good starting point. Two minor, non-blocking suggestions below.

In terms of LHS navigation hierarchy - I was considering where else `Security` may fit..and I think the only way was to integrate it within the existing R&D, IT and Workplace sections. Which feels disconnected.

So +1 with starting with Security as its own category in the LHS.

**Comment by DSchalla** [comment]
Yes, especially with the Privacy sections I wanted to make sure it's easily discoverable for our customers and contributors. Also as you mentioned it doesn't really fit anywhere else.

Thanks Jason!

**Comment by jasonblais** [comment]
@justinegeffen quick reminder to help with a review :)

**Review by justinegeffen — APPROVED** [review]
This is a great start, thanks Daniel! I'm happy to merge this if that works for you. Then you can iterate on the public pages as needed.

**Comment by DSchalla** [comment]
@justinegeffen Thanks! Lets merge this and we will add pages as we continue to mature our section, thank you!

**Pull request merged**